docs/contracts/liquidity-launchpad/quickstart/example-configuration.md

```diff
@@ -0,0 +1,275 @@
+
+### fundsRecipient
+The `fundsRecipient` parameter is the address that will receive the funds raised during the auction. Again, depending on the implementation, this may be a trusted EOA address or a strategy contract address if the auction is being used to bootstrap a liquidity pool.
+
+### startBlock
+The `startBlock` parameter is the block number at which the auction will start. Once the auction starts, the supply schedule will begin and bids will be accepted. Note that `startBlock` is inclusive so the auction will start exactly on the block specified.
+
+### endBlock
+The `endBlock` parameter is the block number at which the auction will end. Note that `endBlock` is exclusive so the auction will end on the block specified. No more bids are accepted at and after the end block.
+
+### claimBlock
+The `claimBlock` parameter is the block number at which purchased tokens can be claimed. It must be at or after the `endBlock`.
+
+### tickSpacing
+The `tickSpacing` parameter denotes the **minimum** price increment for bids. It is used to prevent users from being outbid by others by infinitesimally small amounts and for gas efficiency in finding new clearing prices. Generally integrators should choose a tick spacing of AT LEAST 1 basis point of the floor price. 1% or 10% is also reasonable.
+
+Bids can only be placed at increments of tickSpacing, but the auction may clear at any price. 
+
+### validationHook
+The `validationHook` parameter is an optional contract that can be used to validate bids before they are accepted. It is called before a bid is accepted and can be used to reject bids that do not meet certain criteria. It must implement the `IValidationHook` interface. Use `address(0)` to opt-out of validation.
+
+### floorPrice
+The `floorPrice` parameter is the starting floor price for the auction. It is the minimum price at which bids will be accepted.
+
+All prices in the auction are represented as the ratio of `currency` to `token`. For example, a floor price with an integer component of 1000 means that 1000 `currency` is required to purchase 1 `token`. Additionally, the price is represented as a Q96 fixed-point number to allow for fractional prices. For more details about the Q-number format please refer to this wikipedia [article](https://en.wikipedia.org/wiki/Q_(number_format)).
+
+Using the above example, a floor price of 1000 would be represented as `1000 << 96` or `1000 * 2^96`.
+
+### requiredCurrencyRaised
+The `requiredCurrencyRaised` parameter is the amount of `currency` required to be raised for the auction to graduate. If the auction does not raise this amount, the auction will not graduate and all bidders will be refunded their initial bid amounts. No tokens will be sold and the `totalSupply` will be swept back to the `tokensRecipient`.
```

Comment:
I know we use "refund" elsewhere but I think that's not the right word. Refund implies users get their funds pushed back to them. I think we should say "...and all bidders will be able to withdraw their initial bid amounts"